### DISCUSSION TITLE
A Request for Performance Improvements and Bug Fixing

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22164] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hello to the great team of Elementor.

First of all, I apologize for bringing this up here, because there is no better place than here. And maybe I don't know.  **I also hope the senior members of Elementor see this because they are the ones who make the decisions.**

I love Elementor and work with it all the time. Because I learned web design with Elementor and have grown up with it for 5 years now. So I always think about its strengths and weaknesses. I know you are thinking about it too, but I want to offer my opinion and suggestions.

There are several updates that Elementor updates are released quickly and provide great and useful features. You also released a roadmap for future updates. This is really great. So Elementor is great at releasing new features and has no problems. This is my opinion.

**But the main problem of Elementor at the moment is the bugs that exist from previous versions or rarely appear in new versions. Team Elementor, this is a serious matter.**

I have repeatedly seen various reports saying that Elementor consumes a lot of resources and makes the Ajax request run very slowly. And other things that you are definitely aware of.

On the crocoblock github, a user reported that the ajax request of the JetEngine plugin in Elementor takes 9 seconds. Many users contributed to its discussion, finally one user decided to build a website similar to Elementor with another page builder. Surprisingly, it took less than 1 second to run Ajax on that page builder and jet engine plugin. **If you want, I can send you the link of that conversation.**
In general, there are small and big bugs in Elementor, which is why many users intend to migrate from Elementor.

### Describe the solution you'd like

I don't know how many more versions are left until version 4. But please, before releasing version 4, dedicate one version only to fixing bugs. **Even if you spend 4 months working on this version, it's worth it. Because Elementor will step in strongly in version 4.**

**Release a beta version only to fix bugs.** This beta version is open for two, three or even four months for users to report the desired bugs. The bugs will be fixed one by one, and beta 1, beta 2, beta 3, etc. will be published until the bug fix version is officially released. Elementor really needs this version to fix basic and even small bugs.

I think if you do something like that, users will definitely appreciate it. Because this progress is forward.

Thank you for taking the time to read this❤

**Comment by pauserratgutierrez** [comment]
I get the point. I also agree on having a full version (or more) dedicated mainly to fix bugs and small necessary things mentioned in github.

**Comment by ayoubkhan558** [comment]
## Problem:
Bugs exist in Elementor that are affecting user experience, including slow Ajax requests and high resource consumption.

## Solution: 
Release a dedicated version solely focused on fixing bugs, even if it takes several months. Open a beta version for users to report bugs and release subsequent beta versions until the final bug-fix version is released. This will improve Elementor's overall quality and user satisfaction.
